.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Upgrade pip
+      run: |
+        python -m pip install --upgrade pip
+
+    - name: Build the manylinux2010 image
+      run: docker compose build manylinux2010
+
+    - name: Build the package for Python ${{ matrix.python-version }}
+      run: |
+        version="${{ matrix.python-version }}"
+        docker compose run -e PYTHON_VERSION=$(echo "$version" | sed 's/\.//') manylinux2010
+
+    - name: Upload wheel artifact for Python ${{ matrix.python-version }}
+      uses: actions/upload-artifact@v3
+      with:
+        name: data-validation-wheel-py${{ matrix.python-version }}
+        path: dist/*.whl
+
+    - name: Install built wheel
+      run: pip install dist/*.whl
```

Comment:
Is this just to check whether the wheel can be installed? If so maybe it makes sense to have a `twine check dist/*`?